dtfs-central-api/src/services/portal/facility-amendment.updatePortalFacilityAmendment.service.test.ts

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable import/first */
```

Comment:
I've had to add this in because `jest.mock` is hoisted above the imports (it behaves differently in ESM/CJS annoyingly).

When I remove this & put the imports at the top it doesnt hoist `mockFindOneUser` to the top with the `jest.mock`
![image](https://github.com/user-attachments/assets/7c8e985d-081e-488d-80e8-4dba997d94a6)

When I try use another technique like `jest.spyOn` I hit other issues like
![image](https://github.com/user-attachments/assets/3a656a92-baff-48b9-9b18-df9ab39de0e9)

I could probably just reassign the function `findOneUser = mockFindOneUser` but I think this could run the risk of causing leaky tests if thats not teared down properly at the end

Reply:
Done - https://ukef-dtfs.atlassian.net/browse/DTFS2-7724